app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
Using a class variable `alive` to track instances of `Animal` can lead to issues if instances are not properly managed, especially when they are deleted or go out of scope. Consider using instance-specific tracking or another method to manage the list of alive animals.